opal/mca/base/mca_base_var.c

```diff
@@ -2199,10 +2200,22 @@ int mca_base_var_dump(int vari, char ***out, mca_base_var_dump_type_t output_typ
             return OPAL_ERR_OUT_OF_RESOURCE;
         }
 
+        char *color_var_name = "", *color_var_value = "";
+        char *color_var_valid_values = "", *color_reset = "";
+
+        if(output_type == MCA_BASE_VAR_DUMP_READABLE_COLOR) {
+            color_var_name = "\033[34m";
```

Comment:
can you add some comments (or better yet, make these defines with descriptive names) on what these colors are?

Reply:
Ok, yes. (this will change up a bit if/when I implement the customization support, so I'll take care of it then)